integration-tests/api/__tests__/admin/product.js

```diff
@@ -46,6 +46,7 @@ describe("/admin/products", () => {
     dbConnection = await initDb({ cwd })
     medusaProcess = await setupServer({
       cwd,
+      env: { MEDUSA_FF_PRODUCT_CATEGORIES: true },
```

Comment:
question: Should we separate this into its own FF test suite?

Reply:
split into 2 for both admin & store